modules/nf-core/clippy/main.nf

```diff
@@ -0,0 +1,40 @@
+process CLIPPY {
+    tag "$meta.id"
+    label "process_high"
```

Comment:
Does it def need high?

Reply:
yes we had issues on large files if set lower